package.json

```diff
@@ -106,11 +107,12 @@
       "@salesforce/plugin-command-reference"
     ],
     "topics": {
-      "force": {
+      "generate": {
         "external": true,
+        "description": "Commands to generate a project, create a function, and more.",
         "subtopics": {
           "cmdt": {
-            "description": "Generate custom metadata types and their records.",
+            "description": "Commands to generate custom metadata types and their records.",
             "subtopics": {
```

Comment:
Managed to test this w/CLI Ref.  Can you remove this subtopic for "record"?  It's not needed. thx!